data/net/minecraft/client/gui/screens/reporting/ChatSelectionScreen.mapping

```diff
@@ -51,4 +53,4 @@ CLASS net/minecraft/client/gui/screens/reporting/ChatSelectionScreen
 				ARG 1 poseStack
 				ARG 2 x
 				ARG 3 y
-				ARG 4 skinLocation
+				ARG 4 textureId
```

Comment:
Debatable hmm

Reply:
@Lanse505 @alcatrazEscapee @sciwhiz12 Any thoughts?

Reply:
Yea no, this should be ``skinLocation`` or alternatively ``senderSkinLocation`` (Just from a quick glance also slightly too verbose)

Reply:
Fixed